sys/net/gnrc/sock/include/gnrc_sock_internal.h

```diff
@@ -37,6 +37,22 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default port range for dynamic source port allocation
+ */
+#define GNRC_SOCK_DYN_PORTRANGE_MIN (49152U)
+#define GNRC_SOCK_DYN_PORTRANGE_MAX (65535U)
```

Comment:
I do understand, but again: which header file would be best? Or should we have a iana.h somewhere to organized such official values. In the end we will come to the point where we need IP address ranges or other values assigned by the IANA